src/_path_wrapper.cpp

```diff
@@ -735,27 +734,19 @@ static PyObject *Py_convert_to_string(PyObject *self, PyObject *args, PyObject *
     CALL_CPP("convert_to_string",
              (status = convert_to_string(
                  path, trans, cliprect, simplify, sketch,
-                 precision, codes, postfix, &buffer,
-                 &buffersize)));
-
-    if (status) {
-        free(buffer);
-        if (status == 1) {
-            PyErr_SetString(PyExc_MemoryError, "Memory error");
-        } else if (status == 2) {
-            PyErr_SetString(PyExc_ValueError, "Malformed path codes");
-        }
+                 precision, codes, postfix, buffer)));
+
+    if (!status) {
+        PyErr_SetString(PyExc_ValueError, "Malformed path codes");
         return NULL;
     }
 
-    if (buffersize == 0) {
+    if (buffer.size() == 0) {
```

Comment:
I think this if clause can just be deleted?

Reply:
Removed.